npm install fails for bcrypt

`bcrypt@0.8.7` seems to work, any reason for the [strict requirement for `bcrypt@0.8.5`](https://github.com/shoreditch-ops/artillery-core/blob/5343e19cbd3d0de1a47ae0aabab0cb49f15ed025/package.json#L40)?

```
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/bin/iojs" "/usr/local/bin/npm" "i"
npm ERR! node v6.2.2
npm ERR! npm  v3.9.5
npm ERR! code ELIFECYCLE

npm ERR! bcrypt@0.8.5 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bcrypt@0.8.5 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the bcrypt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bcrypt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls bcrypt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/sz/src/shoreditch-ops/artillery-core/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     "ws": "1.0.1"
   },
   "devDependencies": {
-    "bcrypt": "0.8.5",
+    "bcrypt": "^0.8.5",
     "csv-parse": "1.0.1",
     "eslint": "^0.24.0",
     "good": "6.4.0",

